### ISSUE TITLE
when use the real ratio, the model doesn't work

### ISSUE THREAD

**Issue by sanlb** [Goldesel23/Temporal-Ensembling-for-Semi-Supervised-Learning#3]
hi, 
when l use the real ratio: `labeled_batch_fraction = num_labeled_samples / num_train_unlabeled_sample`(it should be num_labeled_samples / (num_train_unlabeled_sample+num_labeled_samples?)), after training about more than 50 epochs, l find that  the both train and val acc is always between 9%-10%. it means that the model doesn't work at all. do you have the same question? besides, train many times your code, the best test acc is only 89% far away from the paper. is it same with you?
thank you.

**Comment by tensorfreitas** [comment]
Hello, 

Is the training loss decreasing? It should be a problem of high learning rate and/or small batch size. The best test accuracy is around 90/91% (or more if you increase the number of labeled samples), but notice I did not tried hyper-parameter tuning.

**Comment by sanlb** [comment]
ok, l get it, thank you again.
